frontend/src/components/AccountReceiveTable.vue

```diff
@@ -97,7 +97,7 @@
               <q-card-section>
                 <div class="row justify-between items-center">
```

Comment:
Getting these on the Receive page
![image](https://user-images.githubusercontent.com/61768337/225681256-02b05a03-3590-404a-9d70-5e89cf6b1332.png)

Reply:
Above image is from Chrome on Goerli, this one is from Firefox on Goerli:
![image](https://user-images.githubusercontent.com/61768337/225681722-6d649a44-ee1a-4fab-b56d-8438d34d42d6.png)

Reply:
I must be doing something wrong cause I don't even see the Send History button under my account 
![image](https://user-images.githubusercontent.com/61768337/225683011-7db553e7-6a77-47e0-96fc-ff79dcdf46e9.png)

Reply:
Yea, the routes change seems to be the issue. I am going to drop a message in Element

Reply:
This is still happening to me @alexkeating on the Receive page while on Chrome, using Metamask Goerli network.

Reply:
![image](https://user-images.githubusercontent.com/61768337/227294338-ef93860b-5ffc-4f64-966d-fa47e0deec58.png)

Reply:
I think I fixed this